ipam_test.go

```diff
@@ -535,6 +535,80 @@ func TestSpace(t *testing.T) {
 	}
 }
 
+func TestConains(t *testing.T) {
+	tests := []struct {
+		Network          string
+		Subnet           string
+		ExpectedContains bool
+	}{
+		// Test 0.
+		{
+			Network:          "10.4.0.0/16",
+			Subnet:           "10.4.0.0/17",
+			ExpectedContains: true,
+		},
+		// Test 1.
+		{
+			Network:          "10.4.0.1/16",
+			Subnet:           "10.4.0.0/17",
+			ExpectedContains: true,
+		},
+		// Test 2.
+		{
+			Network:          "10.4.0.0/16",
+			Subnet:           "10.4.0.0/16",
+			ExpectedContains: true,
+		},
+		// Test 3.
+		{
+			Network:          "10.4.0.0/16",
+			Subnet:           "10.4.20.0/16",
+			ExpectedContains: true,
+		},
+		// Test 4.
+		{
+			Network:          "10.4.0.0/16",
+			Subnet:           "10.4.20.0/17",
+			ExpectedContains: true,
+		},
+		// Test 5.
+		{
+			Network:          "10.4.0.0/16",
+			Subnet:           "10.4.0.0/15",
+			ExpectedContains: false,
+		},
+		// Test 6.
+		{
+			Network:          "10.4.0.0/16",
+			Subnet:           "10.3.0.0/16",
+			ExpectedContains: false,
+		},
+		// Test 7.
+		{
+			Network:          "10.4.0.0/16",
+			Subnet:           "10.3.0.0/32",
+			ExpectedContains: false,
+		},
+	}
+
+	for i, tc := range tests {
```

Comment:
Would be good to follow [fmt guide](https://github.com/giantswarm/fmt/blob/master/go/unit_tests.md) here and run every test case using `t.Run` along with this we can also have test name comment (e.g. `// Test1.`) as part of test cases struct.

Reply:
Done. Other tests need to be cleaned in a separate round.